### PR TITLE
[BRD-480] Replace HashWithIndifferentAccess

### PR DESCRIPTION
**Pull request by valikos**
Replace HashWithIndifferentAccess with ActiveSupport::HashWithIndifferentAccess
Belongs to: https://github.com/lumoslabs/lumos_rails/pull/10836

### CODE REVIEW

File: lib/changes_are_logged/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ChangesAreLogged
-  VERSION = '1.2.0'
+  VERSION = '1.2.1'
```
Comment (idanci) [inline lib/changes_are_logged/version.rb:2]: Can you also update changelog?

**Review by andriimosin — CHANGES_REQUESTED** [review]
Looks great, but please, let's make sure we didn't forget to update the Changelog

File: lib/changes_are_logged/version.rb (hunk 1 above)
Comment (valikos) [reply lib/changes_are_logged/version.rb:2]: Done.

**Review by andriimosin — APPROVED** [review]
Awesome! Thank you!

**Pull request merged**